Can't build from AUR

Hi,

I'm trying to install Zyn Fusion, but when it build the dependecie mruby-zest-build, I have this trouble :

```
/usr/bin/ld : /tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(nvg_context.o):/tmp/makepkg/zyn-fusion/src/mruby-zest-build/deps/mruby-nanovg/src/nvg_context.c:217 : définitions multiples de « mrb_nvg_context_type »; /tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(gem.o):/tmp/makepkg/zyn-fusion/src/mruby-zest-build/src/mruby-widget-lib/src/gem.c:293 : défini pour la première fois ici
collect2: erreur: ld a retourné le statut de sortie 1
/usr/bin/ld : /tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(nvg_context.o):/tmp/makepkg/zyn-fusion/src/mruby-zest-build/deps/mruby-nanovg/src/nvg_context.c:217 : définitions multiples de « mrb_nvg_context_type »; /tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(gem.o):/tmp/makepkg/zyn-fusion/src/mruby-zest-build/src/mruby-widget-lib/src/gem.c:293 : défini pour la première fois ici
collect2: erreur: ld a retourné le statut de sortie 1
rake aborted!
Command failed with status (1): [gcc -Wl,-O1,--sort-common,--as-needed,-z,r...]
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:31:in `_run'
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:36:in `rescue in _run'
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:32:in `_run'
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:183:in `run'
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/Rakefile:72:in `block (4 levels) in <top (required)>'
Caused by:
Command failed with status (1): ["gcc" -Wl,-O1,--sort-common,--as-needed,-z...]
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:33:in `_run'
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:183:in `run'
/tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/Rakefile:72:in `block (4 levels) in <top (required)>'
Tasks: TOP => default => all => /tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/bin/mruby => /tmp/makepkg/zyn-fusion/src/mruby-zest-build/mruby/build/host/bin/mruby
(See full trace by running task with --trace)
make: *** [Makefile:13 : all] Erreur 1
==> ERREUR : Une erreur s’est produite dans build().
    Abandon…
Error making: zyn-fusion
```

Do you have a solution ?

Thank you

Same in Fedora (RPM):

```
/usr/share/ruby/fileutils.rb:426: warning: The called method `cp' is defined here
CC    mrbgems/mruby-bin-mruby/tools/mruby/mruby.c -> build/host/mrbgems/mruby-bin-mruby/tools/mruby/mruby.o
LD    build/host/bin/mruby 
/usr/bin/ld: /home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/build/host/lib/libmruby.a(gem.o):/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/src/mruby-widget-lib/src/gem.c:293: multiple definition of `mrb_nvg_context_type'; /home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/build/host/lib/libmruby.a(nvg_context.o):(.data.rel.ro.local+0x0): first defined here
collect2: error: ld returned 1 exit status
/usr/bin/ld: /home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/build/host/lib/libmruby.a(gem.o):/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/src/mruby-widget-lib/src/gem.c:293: multiple definition of `mrb_nvg_context_type'; /home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/build/host/lib/libmruby.a(nvg_context.o):(.data.rel.ro.local+0x0): first defined here
collect2: error: ld returned 1 exit status
rake aborted!
Command failed with status (1): [gcc -L/home/bruno2/l/dev/zyn-fusion-build/...]
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/lib/mruby/build/command.rb:31:in `_run'
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/lib/mruby/build/command.rb:36:in `rescue in _run'
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/lib/mruby/build/command.rb:32:in `_run'
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/lib/mruby/build/command.rb:210:in `run'
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/Rakefile:79:in `block (4 levels) in <top (required)>'
/usr/share/gems/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'

Caused by:
Command failed with status (1): [gcc -L/home/bruno2/l/dev/zyn-fusion-build/...]
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/lib/mruby/build/command.rb:33:in `_run'
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/lib/mruby/build/command.rb:210:in `run'
/home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/Rakefile:79:in `block (4 levels) in <top (required)>'
/usr/share/gems/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => default => all => /home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/bin/mruby => /home/bruno2/l/dev/zyn-fusion-build/mruby-zest-build/mruby/build/host/bin/mruby
(See full trace by running task with --trace)
make: *** [Makefile:13: all] Error 1
ERROR: 'make' failed

```

I'm trying to build zyn-fusion from the AUR, but it fails after some time and gives this output

`ar: creating /home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/build/host-debug/lib/libmruby.a
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/Rakefile:45: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/lib/ruby/2.7.0/fileutils.rb:595: warning: The called method `rm_f' is defined here
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/Rakefile:46: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/lib/ruby/2.7.0/fileutils.rb:426: warning: The called method `cp' is defined here
CC    mrbgems/mruby-bin-mruby/tools/mruby/mruby.c -> build/host/mrbgems/mruby-bin-mruby/tools/mruby/mruby.o
LD    build/host/bin/mruby 
/usr/bin/ld: /home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(nvg_context.o):(.data.rel.ro.local+0x0): multiple definition of `mrb_nvg_context_type'; /home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(gem.o):/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/src/mruby-widget-lib/src/gem.c:293: first defined here
collect2: error: ld returned 1 exit status
/usr/bin/ld: /home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(nvg_context.o):(.data.rel.ro.local+0x0): multiple definition of `mrb_nvg_context_type'; /home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/build/host/lib/libmruby.a(gem.o):/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/src/mruby-widget-lib/src/gem.c:293: first defined here
collect2: error: ld returned 1 exit status
rake aborted!
Command failed with status (1): [gcc -Wl,-O1,--sort-common,--as-needed,-z,r...]
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:31:in `_run'
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:36:in `rescue in _run'
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:32:in `_run'
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:183:in `run'
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/Rakefile:72:in `block (4 levels) in <top (required)>'

Caused by:
Command failed with status (1): ["gcc" -Wl,-O1,--sort-common,--as-needed,-z...]
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:33:in `_run'
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/tasks/mruby_build_commands.rake:183:in `run'
/home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/Rakefile:72:in `block (4 levels) in <top (required)>'
Tasks: TOP => default => all => /home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/bin/mruby => /home/apoorv/.cache/yay/zyn-fusion/src/mruby-zest-build/mruby/build/host/bin/mruby
(See full trace by running task with --trace)
make: *** [Makefile:13: all] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
Error making: zyn-fusion
`
please help, as i really in need for this plugin my projects depends on this plugin.

--- a/src/mruby-widget-lib/src/gem.c
+++ b/src/mruby-widget-lib/src/gem.c
@@ -290,7 +290,7 @@ mrb_fbo_deselect(mrb_state *mrb, mrb_value self)
     return self;
 }
 
-const struct mrb_data_type mrb_nvg_context_type;
+extern const struct mrb_data_type mrb_nvg_context_type;
 typedef void NVGcontext;
 int nvglCreateImageFromHandleGL2(NVGcontext* ctx, GLuint textureId, int w, int h, int imageFlags);
 

